File: 3986-number-of-elapsed-seconds-between-two-times/3986-number-of-elapsed-seconds-between-two-times.java
class Solution {
    public int secondsBetweenTimes(String startTime, String endTime) {
        return toSeconds(endTime) - toSeconds(startTime);
    }

    private int toSeconds(String time) {
        int h = Integer.parseInt(time.substring(0, 2));
        int m = Integer.parseInt(time.substring(3, 5));
        int s = Integer.parseInt(time.substring(6, 8));

        return h * 3600 + m * 60 + s;
    }
}

File: 3986-number-of-elapsed-seconds-between-two-times/3986-number-of-elapsed-seconds-between-two-times.py
class Solution(object):
    def secondsBetweenTimes(self, startTime, endTime):
        """
        :type startTime: str
        :type endTime: str
        :rtype: int
        """

        def toSeconds(t):
            h = int(t[:2])
            m = int(t[3:5])
            s = int(t[6:])
            return h * 3600 + m * 60 + s

        return toSeconds(endTime) - toSeconds(startTime)